src/core/lib/experiments/experiments.yaml

```diff
@@ -66,6 +66,7 @@
   expiry: 2025/06/01
   owner: ctiller@google.com
   test_tags: [core_end2end_test]
+  allow_in_fuzzing_config: false
```

Comment:
Why was this needed?

Reply:
Normally we allow the fuzzer to choose experiments to mix and match in end2end tests, and that helps flush out bad interactions early.

In this case, I need to mutate the set of end2end test configurations based on experiment state, and so that state needs to be constant across fuzzer runs - which means we can't allow the fuzzer to choose those values to flip.